app/server/middleware/sanitizeInputs.js

```diff
@@ -30,7 +29,7 @@ const asyncHandler = require("express-async-handler");
  * Does NOT work with req.params, BUT params are passed as a string
  * instead of an object so there should be no worries of injection.
  */
-sanitize = (input) => {
+const sanitize = (input) => {
   if (input instanceof Object) {
     Object.keys(input).forEach((key) => {
       if (/^\$/.test(key)) {
```

Comment:
What is this trying to sanitize? Right now that regex only catches anything that starts with '$'.